WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -33,32 +34,32 @@ regtest.rpcport = 18443
 regtest.rpcuser = 7c9b6473600fbc9be1120ae79f1622f42c32e5c78d
```

Comment:
is this the default password and user, or one randomly specified by @molnard, should the user change this password?